bears/r/FormatRBear.py

```diff
@@ -8,14 +9,55 @@
                                         'to work.')
 class FormatRBear:
+
     """
-    This bear checks and corrects formatting of R Code using known formatR
+    Check and correct formatting of R Code using known formatR
     utility.
     """
     LANGUAGES = "R"
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
-        rcode = 'formatR::tidy_source(source ="' + \
-                                      escape(filename, '"\\') + '")'
+    def create_arguments(filename, file, config_file,
+                         r_keep_comments: bool=True,
+                         r_keep_blank_lines: bool=True,
+                         r_braces_on_next_line: bool=False,
+                         r_use_arrows: bool=False,
+                         tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
+                         r_max_expression_length: int=20):
+        """
+        :param r_keep_comments:         Use to determine whether comments are
+                                        kept or not.
+        :param r_keep_blank_lines:      Use to determine whether blank lines
+                                        are kept or not.
+        :param r_braces_on_next_line:   Use to determine wheter a brace should
+                                        be placed on the next line.
+        :param r_use_arrows:            Use to determine if either the assign
+                                        operator ``=`` or the arrow ``<-``
+                                        should be used.
+        :param tab_width:               Number of space for indentation.
+        :param r_max_expression_length: Maximum number of characters for an
+                                        expression. Note (different from the
+                                        maximum number of characters for
+                                        a line).
+        """
+        rcode = ('formatR::tidy_source('
+                 'source="' + escape(filename, '"\\') + '",'
+                 'comment={r_keep_comments},'
+                 'blank={r_keep_blank_lines},'
+                 'arrow={r_use_arrows},'
+                 'width.cutoff={r_max_expression_length},'
+                 'brace.newline={r_braces_on_next_line},'
+                 'indent={tab_width}'
+                 ')'.format(tab_width=tab_width,
+                            r_keep_comments=_map_to_r_bool(r_keep_comments),
+                            r_keep_blank_lines=_map_to_r_bool(
+                                                        r_keep_blank_lines),
+                            r_use_arrows=_map_to_r_bool(r_use_arrows),
+                            r_braces_on_next_line=_map_to_r_bool(
+                                r_braces_on_next_line),
+                            r_max_expression_length=r_max_expression_length))
         return ('-e', 'library(formatR)', '-e', rcode)
+
+
+def _map_to_r_bool(py_bool):
+    return 'TRUE' if py_bool else 'FALSE'
```

Comment:
just preference, I would align it above the class file, after a class ends normally my intention is also thinking the file ends :) but if you want you can keep it :+1: